src/utils/aosheader.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef AOSHEADER_HPP
+#define AOSHEADER_HPP
+
+#include <aos/common/tools/array.hpp>
+#include <aos/common/tools/error.hpp>
+#include <aosprotocol.h>
+
+#include "checksum.hpp"
+
+static aos::RetWithError<AosProtocolHeader> PrepareHeader(uint32_t port, const aos::Array<uint8_t>& data)
```

Comment:
It should be private method of channel manager

Reply:
Done